src/components/Resources/FilterButtons.tsx

```diff
@@ -12,18 +12,19 @@ export const FilterButtons = () => {
 
 const HeartedButton = () => {
     return (
-        <div className="flex w-1/2 flex-row items-center rounded bg-orange-100 p-2 text-slate-900 hover:bg-orange-200">
+        <div className="flex w-1/2 flex-row items-center rounded bg-orange-100 p-2 text-slate-900 hover:bg-orange-200 md:w-full">
             <div className="px-2">
                 <FaHeart />
             </div>
-            Hearted
+            <span className="block md:hidden">Hearted</span>
+            <span className="hidden md:block">Hearted Resources</span>
```

Comment:
Does the `sm` breakpoint still give enough space for the full word? and would we consider just ellipses to truncate text as a more flexible solution?

Reply:
Consistently labeled both 'Hearted resources', I just thought it looked a little crowded on the mobile breakpoint